beacon_node/http_api/src/publish_blocks.rs

```diff
@@ -178,10 +199,6 @@ pub async fn publish_block<T: BeaconChainTypes, B: IntoGossipVerifiedBlock<T>>(
         Err(BlockError::Slashable) => Err(warp_utils::reject::custom_bad_request(
             "proposal for this slot and proposer has already been seen".to_string(),
         )),
-        Err(BlockError::BlockIsAlreadyKnown) => {
```

Comment:
Is it no longer possible to hit this error here? I've looked around and don't think it's possible but just wanted to double check.

Reply:
Yeah, `process_block` should be incapable of returning this error, as it only happens in gossip block verification.